### DISCUSSION TITLE
New `Folders` category to mirror the filesystem tree

### DISCUSSION THREAD

**Discussion by nagyrobi** [music-assistant/support#413] [Feature requests and ideas]
Follow up from https://discord.com/channels/753947050995089438/753947050995089442/980834819053330532

For the filesystem provider, create a new category in Music Assistant `Folders` where one could view the media organized in a tree corresponding to how they are on the filesystem. Would be useful for cases when ID3/metadata is not consistent.
Media should be organized alphabetically by file/folder name, (for folders containing files and subfolders, put files first).

Each folder could be treated as a queque actually, containing all media items from all the subfolders, so one could play the contents of the folder as a playlist.

To discuss how to save the order of the items in the queue (eg one may want to sort by date files of a certain folder, or may want to adjust order by hand).

**Comment by erkr** [comment]
I move this request to the discussions, thanks for filing.

**Comment by OzGav** [comment]
This is a v1 suggestion and since then the browse option has been added.
